server/routerlicious/packages/services-core/src/database.ts

```diff
@@ -33,20 +33,73 @@ export interface IDatabaseManager {
 }
 
 export interface ICollection<T> {
+    /**
+     * Finds queries in the database and returns sorted result.
+     *
+     * @param query - data we want to find
+     * @param sort - the value used to sort data.
+     *
+     */
     find(query: any, sort: any): Promise<T[]>;
 
+    /**
+     * Finds one query in the database, returns its value
+     *
+     *  @param query - data we want to find
+     */
     findOne(query: any): Promise<T>;
 
+    /**
+     * Returns all values in the database
+     */
     findAll(): Promise<T[]>;
 
+    /**
+     * Finds query in the database and returns its value.
+     * Insert the value if query was not found.
+     *
+     * @param query - data we want to find
+     * @param value - data to insert to the database if we cannot find query
+     */
     findOrCreate(query: any, value: T): Promise<{ value: T, existing: boolean }>;
 
+    /**
+     *
+     * Finds the query in the db. If it exists, update the value to set.
+     * Throws if query cannot be found.
+     *
+     *  @param filter - data we want to find
+     *  @param set - new values to change to
+     *  @param addToSet
```

Comment:
What does this param do?

Reply:
i dont know...it's not used anywhere except in the mongodb.ts, where it's also ambiguous what it's used for